libs/sdk-ui-dashboard/src/model/store/context/contextState.ts

```diff
@@ -0,0 +1,13 @@
+// (C) 2021-2025 GoodData Corporation
+import { PrivateDashboardContext } from "../../types/commonTypes.js";
+
+/**
+ * @beta
+ */
+export interface ContextState {
+    existingExportTransformFn: PrivateDashboardContext["existingExportTransformFn"];
+}
+
+export const contextInitialState: ContextState = {
+    existingExportTransformFn: undefined,
```

Comment:
Do we really want this to be in the state? Maybe React Context would be better  for this? Also "context state" is very vague name.